test/unit/calculators/benefit_cap_calculator_data_query_test.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "../../test_helper"
+
+module SmartAnswer::Calculators
+  class BenefitCapCalculatorDataQueryTest < ActiveSupport::TestCase
+    context BenefitCapCalculatorDataQuery do
+      setup do
+        @query = BenefitCapCalculatorDataQuery.new
+      end
+
+      context "benefit_cap_data" do
+        context "family types" do
+          should "contain value for single person" do
```

Comment:
Assuming the rename of `#data` to `#weekly_benefit_cap` (mentioned above) then these tests could be renamed from `"family types" should "contain value for single person"` to `"#weekly_benefit_cap" should "be £350 for a single person"`.